tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -54,9 +54,9 @@ int main(int argc, char** argv) {
         std::cout << "ERROR : please provide the platform test path to -p option\n";
         return EXIT_FAILURE;
     }
+    std::string binaryFile = test_path + b_file;
```

Comment:
Please use boost::filesystem::path to join a path and file.  The solution provided above will be problematic across multiple OSs.